catkin.options

```diff
@@ -0,0 +1 @@
+-DMY_CMAKE_CXX_FLAGS=--coverage -g -O0
```

Comment:
By putting this in a different file the options are only used when building in travis(though I could probably do this in the robosherlock.cmake as well). The options have been taken from a sample project, i thought it would be good to explicitly set them even if they may be default.